### PR TITLE
Implement activity logs photos component

### PR DESCRIPTION
**Pull request by Parveshdhull**
### Summary

PR implements activity logs photos component.

related issue: https://github.com/status-im/status-mobile/issues/16054, https://github.com/status-im/status-mobile/pull/16707#issuecomment-1640014061
Designs - https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=13336-165824&mode=design&t=KadcnKQ0KDgiTGqv-0

status: ready

### CODE REVIEW

File: src/status_im2/contexts/quo_preview/list_items/preview_lists.cljs [hunk 1]
```diff
@@ -40,9 +40,9 @@
   [{:source (resources/get-mock-image :photo1)}
    {:source (resources/get-mock-image :photo2)}
    {:source (resources/get-mock-image :photo3)}
-   {:source (resources/get-mock-image :photo4)}
-   {:source (resources/get-mock-image :photo5)}
-   {:source (resources/get-mock-image :photo6)}])
```
Comment (Parveshdhull) [inline src/status_im2/contexts/quo_preview/list_items/preview_lists.cljs:45]: These photos not available anymore

File: src/quo2/components/notifications/activity_logs_photos/view.cljs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+(ns quo2.components.notifications.activity-logs-photos.view
+  (:require [react-native.core :as rn]
+            [quo2.components.notifications.activity-logs-photos.style :as style]))
+
+(defn view
+  [photos]
```
Comment (J-Son89) [inline src/quo2/components/notifications/activity_logs_photos/view.cljs]: nit: should we use a map for first arg as a standard for all our quo2 components? It's quite react style to do that 👍

**Review by ilmotta — APPROVED** [review]
LGTM!

**Review by J-Son89 — APPROVED** [review]

**Review by smohamedjavid — APPROVED** [review]

**Review by Francesca-G — APPROVED** [review]
The component looks good to me :)

**Pull request merged**